Update video audio types mp3,mkv,mp4,webm

ahmedmohiduet opened the pull request:
Getting results :
application/octet-stream
application/octet-stream
application/octet-stream
application/octet-stream

for all of my codes:
System.out.println(MimetypesFileTypeMap.getDefaultFileTypeMap().getContentType("/usr/lo.cal/myfile.mp3"));
System.out.println(MimetypesFileTypeMap.getDefaultFileTypeMap().getContentType("/usr/lo.cal/myfile.mkv"));
System.out.println(MimetypesFileTypeMap.getDefaultFileTypeMap().getContentType("/usr/lo.cal/myfile.mp4"));
System.out.println(MimetypesFileTypeMap.getDefaultFileTypeMap().getContentType("/usr/lo.cal/myfile.webm"));

ahmedmohiduet committed:
Update video audio types mp3,mkv,mp4,webm

Getting results :
application/octet-stream
application/octet-stream
application/octet-stream
application/octet-stream

for all of my codes:
System.out.println(MimetypesFileTypeMap.getDefaultFileTypeMap().getContentType("/usr/lo.cal/myfile.mp3"));
System.out.println(MimetypesFileTypeMap.getDefaultFileTypeMap().getContentType("/usr/lo.cal/myfile.mkv"));
System.out.println(MimetypesFileTypeMap.getDefaultFileTypeMap().getContentType("/usr/lo.cal/myfile.mp4"));
System.out.println(MimetypesFileTypeMap.getDefaultFileTypeMap().getContentType("/usr/lo.cal/myfile.webm"));

buildhive commented:
[Kohsuke Kawaguchi » javax.activation #2](https://buildhive.cloudbees.com/job/kohsuke/job/javax.activation/2/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

ahmedmohiduet commented:
Hi kohsuke, can this repository be available from maven central repository?

ahmedmohiduet committed:
Update video audio types mp3,mkv,mp4,webm, update with capital extensions

Getting results :
application/octet-stream
application/octet-stream
application/octet-stream
application/octet-stream

for all of my codes:
System.out.println(MimetypesFileTypeMap.getDefaultFileTypeMap().getContentType("/usr/lo.cal/myfile.mp3"));
System.out.println(MimetypesFileTypeMap.getDefaultFileTypeMap().getContentType("/usr/lo.cal/myfile.mkv"));
System.out.println(MimetypesFileTypeMap.getDefaultFileTypeMap().getContentType("/usr/lo.cal/myfile.mp4"));
System.out.println(MimetypesFileTypeMap.getDefaultFileTypeMap().getContentType("/usr/lo.cal/myfile.webm"));

buildhive commented:
[Kohsuke Kawaguchi » javax.activation #3](https://buildhive.cloudbees.com/job/kohsuke/job/javax.activation/3/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

ahmedmohiduet committed:
Updated content types for flv, x-flv

buildhive commented:
[Kohsuke Kawaguchi » javax.activation #4](https://buildhive.cloudbees.com/job/kohsuke/job/javax.activation/4/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

kohsuke commented:
This is a dormant repository and since I left Sun I don't think I can publish bits to anywhere official.

So I'm going to just close this PR.

(BTW it doesn't make sense to list both upper case and lower case extensions. The file only needs to list lower case, and code should be normalizing the extension to lower-case before checking the file.)
